app/ui/view_nano.c

```diff
@@ -70,10 +70,10 @@ void view_error_show() {
     view_error_show_impl();
 }
 
-void view_custom_error_show(const char *title, const char *statusString) {
-    snprintf(viewdata.key, MAX_CHARS_PER_KEY_LINE, "%s", title);
-    snprintf(viewdata.value, MAX_CHARS_PER_VALUE1_LINE, "%s", statusString);
-    view_error_show_impl();
+void view_custom_error_show(char *upper, char *lower) {
+    snprintf(viewdata.key, MAX_CHARS_PER_KEY_LINE, "%s", upper);
+    snprintf(viewdata.value, MAX_CHARS_PER_KEY_LINE, "%s", lower);
```

Comment:
That `MAX_CHARS_PER_KEY_LINE` it's max title chars (the first line, the one in bold). Should be `MAX_CHARS_PER_VALUE1_LINE`